radware-alteon/info.json

```diff
@@ -0,0 +1,246 @@
+{
+    "name": "radwareAlteon",
+    "version": "1.0.8",
+    "label": "Radware Alteon",
+    "description": "",
+    "publisher": "Danny",
+    "icon_small_name": "RDWR logo small.png",
+    "icon_large_name": "RDWR logo large.png",
+    "category": "",
+    "configuration": {
+        "fields": [
+            {
+                "title": "URL",
+                "type": "text",
+                "name": "uRL",
+                "required": true,
+                "visible": true,
+                "editable": true,
+                "value": "https://192.168.1.1",
+                "tooltip": "the api url for the Radware Alteon"
+            },
+            {
+                "title": "b64EncodedAuth",
+                "type": "password",
+                "name": "b64EncodedAuth",
+                "required": true,
+                "visible": true,
+                "editable": true,
+                "value": null
+            },
+            {
+                "title": "Username",
+                "type": "text",
+                "name": "username",
+                "required": false,
+                "visible": false,
+                "editable": true
+            },
+            {
+                "title": "Password",
+                "type": "password",
+                "name": "password",
+                "required": false,
+                "visible": false,
+                "editable": true
+            },
+            {
+                "title": "UserPassEncoding",
+                "type": "text",
+                "name": "userPassEncoding",
+                "required": true,
+                "visible": true,
+                "editable": true,
+                "value": "utf-8"
+            },
+            {
+                "title": "Verify SSL",
+                "type": "checkbox",
+                "name": "verify_ssl",
+                "required": false,
+                "visible": true,
+                "editable": true
+            }
+        ]
+    },
+    "operations": [
+        {
+            "operation": "view_table",
+            "title": "View Table",
+            "description": "ipAclNewCfgTable.html: View Table",
+            "enabled": true,
+            "category": "utilities",
+            "annotation": "",
+            "output_schema": {
+                "result": "",
+                "api_data": ""
+            },
+            "parameters": [],
+            "open": false
+        },
+        {
+            "operation": "view_table_element",
+            "title": "View Table Element",
+            "description": "ipAclNewCfgTable.html: View Table Element",
+            "category": "utilities",
+            "annotation": "",
+            "output_schema": {
+                "result": "",
+                "api_data": ""
+            },
+            "enabled": true,
+            "parameters": [
+                {
+                    "title": "indx",
```

Comment:
```suggestion
                    "title": "Index",
```

Reply:
cannot change this due to the naming due to it has to match the document of the .html file